modulefiles/tasks/derecho/get_obs.local.lua

```diff
@@ -0,0 +1 @@
+load("run_vx.local")
```

Comment:
We now need to load the run_vx module for get_obs tasks in order to access the METplus templating routine.